src/controller/CommissioningDelegate.h

```diff
@@ -407,6 +416,7 @@ class CommissioningParameters
         nullptr; // Delegate to handle device attestation failures during commissioning
     Optional<bool> mAttemptWiFiNetworkScan;
     Optional<bool> mAttemptThreadNetworkScan; // This automatically gets set to false when a ThreadOperationalDataset is set
+    Optional<bool> mPASEOnlyCommissioning;
```

Comment:
I think this would be clearer as mSkipCommissioningComplete or so.

Reply:
updated name